src/util.cpp

```diff
@@ -81,10 +81,12 @@
 #endif
 
 #include <boost/algorithm/string/case_conv.hpp> // for to_lower()
+#include <boost/algorithm/string/join.hpp>
```

Comment:
The commit message doesn't match with what happens here...you probably mean "add back used includes" :)
What about just squashing this into the previous commit?

Reply:
done